modules/ebi-metagenomics/mapseq/meta.yml

```diff
@@ -0,0 +1,56 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "mapseq"
+description: Perform taxonomic classification of rRNA reads using reference databases
+keywords:
+  - taxonomy
+  - reads
+  - otu
```

Comment:
shouldn't be `OTU`?

Reply:
I thought keywords were always lower-case? I found some examples of `OTU` being written in lower case in a few nf-core modules (though admittedly written by a lot of the same people):

https://github.com/nf-core/modules/blob/master/modules/nf-core/taxpasta/standardise/meta.yml
https://github.com/nf-core/modules/blob/master/modules/nf-core/taxpasta/merge/meta.yml
https://github.com/nf-core/modules/blob/master/modules/nf-core/motus/merge/meta.yml

Reply:
then... don't pay attention to me. I'm a big fan of lowercasing everything